spec/support/vcr_cassettes/decision_review/HLR-CREATE-RESPONSE-503_V1.yml

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Wasn't sure how to best simulate the 503, so I just copied the 200 response and changed the status, message, and cleaned up the response body to simulate the error getting raised for the test I added 🤷‍♀️ hope that's okay